config/manifests/bases/placement-operator.clusterserviceversion.yaml

```diff
@@ -6,8 +6,12 @@ metadata:
     capabilities: Basic Install
     features.operators.openshift.io/disconnected: "true"
     features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "true"
```

Comment:
this one we were a littel unsure about.
the only tls endpoint the operator has is the webhook.

if this is referring to the deployment of placement then in theory you can use the service override to tweak the tls profile of the relevant openshfit route that is used to expose it so in that context it would be true.

did we get clarity on if that counted in this context.

Reply:
for a now there is no clarity with tls so I put this pr on hold